INSTALL

```diff
@@ -1,288 +1,368 @@
+'--help'
+'-h'
+     Print a summary of all of the options to 'configure', and exit.
 
-They installation can in addition be controlled by several environment variables:
+'--help=short'
+'--help=recursive'
+     Print a summary of the options unique to this package's
+     'configure', and exit.  The 'short' variant lists options used only
+     in the top level, while the 'recursive' variant lists options also
+     present in any nested packages.
 
-NMZ_COMPILER: overrides the standard choice of compiler (g++ on Linux, clang++ on Mac)
-NMZ_SHARED: if set, only shared libraries will be built; binaries depend on all of them.
-NMZ_PREFIX: if set defines the path for make install; by default it is "local" in the Normaliz directory
-NMZ_COMPILER: overrides the standard choice of compiler (g++ on Linux, clang++ on Mac)
-WITH_GMP: sets the path to the GMP files
-NMZ_OPT_DIR: sets the directory to which the sources of the optional libraries are downloaded
-NMZ_MAC_STATIC: if set a Mac binary is built with dependency on libomp.dylib relative to the binary
-NO_OPENMP: if set blocks OpenMP in the compilation of Normaliz (and e-antic)
+'--version'
+'-V'
+     Print the version of Autoconf used to generate the 'configure'
+     script, and exit.
 
-
-GMP_INSTALLDIR: sets the path to the GMP files
-
-NMZ_OPT_DIR: sets the directory to which the sources of the optional libraries are downloaded;
-  by default it is the subdirectory "nmz_opt_lib"
-
-NMZ_MAC_STATIC: if set, a Mac binary is built with dependency on libomp.dylib relative to the binary
-
-EXTRA_FLAGS: One can further control the build process by setting variables of configure using this
-   variable, the contents of which are appended to the call to the configure script; see
-
-   ./configure --help
-
-Have a look at the install scripts.
-
-*****************************
-****   TROUBLESHOOTING   ****
-*****************************
-
-In order to produce the semistatic binary (Linux and MacOS) install_normaliz.sh must make
-the dynamic libraries invisble: after copying them to the subdirectory save_dynamic, 
-removes them /local/lib and restores them after the build.
-
-If you should encounter any problems at this point, manual intervention is required.
-Since the dynamic libraries have been copied to /local/lib/save_dynamic before
-deletion, you can copy them back to /local/lib. If the worst comes to the worst, you
-must re-install the optional packages.
-
-*********************
-****     SCIP    ****
-*********************
-
-To build SCIP:
-
-* download the scipoptsuite at http://scip.zib.de/ . 
-
-   Notice that SCIP
-   is not distributed under GPL, but the ZIB Academic License
-   (http://scip.zib.de/academic.txt).
-   
-* Unpack it and then compile it with
- 
-   make ZLIB=false GMP=false READLINE=false scipoptlib
-   
-   (To compiling scip for 32bit systems, add also ARCH=x86 to the make.)
-   
-   If your C++ compiler is too old, you may need to add
-   CXXFLAGS="-std=c++0x -fPIC"
```

Comment:
So you have removed all these instructions, by replacing INSTALL with the stock GNU INSTALL file. That seems like a bad idea to me.

Reply:
That was not intentional.

Reply:
So, is there now a need still for a custom INSTALL?